presto-benchmark-runner/src/main/java/com/facebook/presto/benchmark/event/BenchmarkSuiteEvent.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.benchmark.event;
+
+import com.facebook.airlift.event.client.EventField;
+import com.facebook.airlift.event.client.EventType;
+
+import javax.annotation.concurrent.Immutable;
+
+import java.util.Optional;
+
+import static com.facebook.presto.benchmark.event.BenchmarkQueryEvent.EventStatus;
+import static java.util.Objects.requireNonNull;
+
+@Immutable
+@EventType("BenchmarkSuite")
+public class BenchmarkSuiteEvent
+{
+    private final String status;
+    private final int successfulPhases;
+    private String errorMessage;
+
+    public BenchmarkSuiteEvent(EventStatus status, int successfulPhases, Optional<String> errorMessage)
```

Comment:
Use a separate class to represent the status, instead of modifying this in the following commit.

Reply:
One argument per line, to be consistent with `BenchmarkPhaseEvent`

Reply:
Create 2 factory constructors:
```
public static BenchmarkSuiteEvent succeeded(String name); 
public static BenchmarkSuiteEvent failed(String name, String errorMessage); 
```